src/metabase/util/fonts.clj

```diff
@@ -0,0 +1,41 @@
+(ns metabase.util.fonts
+  "font loading functionality."
+  (:require [clojure.string :as str]
+            [clojure.tools.logging :as log]
+            [metabase.util.files :as u.files]))
+
+(def font-path
```

Comment:
Consider making this variable private.

Reply:
Good point. I use this in font_test.clj, is there a quick and easy way to use a private def in another namespace?

I know `#'fonts/private-fn` works for functions, but I don't think that'll work for a private def.

Reply:
You can use `@#'fonts/private-fn`. `#'fonts/private-fn` is the var; deref-ing it with `@` gets its contents